Fushigi/ui/widgets/CourseScene.cs

```diff
@@ -2658,22 +2748,19 @@ private void SelectLayerToAdd()
                 {
                     for (var i = 0; i < fileteredLayers.Count; i++)
                     {
-                        var layer = i < 2 ? 
-                            fileteredLayers[i]+$" ({mLayersVisibility.Count(x => Regex.Replace(x.Key, @"\\d+", "").Contains(fileteredLayers[i]))}/10)":
-                            fileteredLayers[i];
+                        layerCount = mLayersVisibility.Count(x => RxLetters.Replace(x.Key, "") == fileteredLayers[i]);
+                        var layer = i < 2 ? $"{fileteredLayers[i]} ({layerCount}/{MaxLayerCount})" : fileteredLayers[i];
```

Comment:
Can't believe I didn't catch this before, this is REALLY bad code. It doesn't even work rn because you changed the order with the layerTypes but even if it was working, using the filter feature would still break it.
Also it's very hard to understand what the first 2 entries are supposed to be without the context of the previous commits.
Just compare the string directly and save everyone the headache.